makefile

```diff
@@ -11,7 +11,7 @@ ifeq ($(PLATFORM),Darwin)
 endif
 ifeq ($(PLATFORM),Linux)
 	THIRD_PARTY_DIR := third_party_libs/linux
-	SYSTEM_LIBS := -lc -lpthread -lrt -lboost_system -lboost_thread-mt
+	SYSTEM_LIBS := -lc -lpthread -lrt -lboost_system -lboost_thread
```

Comment:
I couldn't build on Linux with the `-mt` there.  The `boost_system` library had the `-mt` suffix removed, so it seems like the right thing to do.